ui-core/src/components/inputs/TolerancePicker.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useRef } from 'react';
+import Input, { InputProps } from './Input';
+import Modal from '../Modal';
+
+const TolerancePicker: React.FC<InputProps> = (inputProps) => {
+  const [showPicker, setShowPicker] = useState(false);
+  const [minusTolerance, setMinusTolerance] = useState('00');
+  const [plusTolerance, setPlusTolerance] = useState('00');
+  const [inputValue, setInputValue] = useState('');
```

Comment:
The inputValue and setInputValue should come from the parent component since this component is only a ui-component (no data manipulation here)